src/web.ts

```diff
@@ -53,7 +80,29 @@ export class RadarPluginWeb extends WebPlugin implements RadarPlugin {
   }
 
   requestLocationPermissions(): void {
-    // not implemented
+    // Not implemented
+  }
+
+  getBeaconsPermissionStatus(): Promise<RadarBeaconsPermissionsCallback> {
+    return new Promise(resolve => {
+      const navigator = window.navigator as any;
+
+      if (typeof navigator === 'undefined' || !navigator.permissions) {
+        resolve({
+          status: 'UNKNOWN'
```

Comment:
should this be `NOT_DETERMINED` to be consistent with the enum for location permissions?

Reply:
This method is almost entirely copied from `getLocationPermissionsStatus`, so it follows that pattern. I agree, this seems counter-intuitive/different than the the native methods though, but just following the exist implementation for now.

Reply:
yeah I guess my only concern is that if a customer using Capacitor is analyzing the distribution of their users' location permissions, then this value might skew the stats if it's not consistent with what our BE expects

Reply:
Agreed. Done.